pom.xml

```diff
@@ -57,80 +55,91 @@
   <scm>
     <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}.git</connection>
     <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}.git</developerConnection>
-    <url>http://github.com/jenkinsci/${project.artifactId}</url>
+    <url>https://github.com/jenkinsci/${project.artifactId}</url>
     <tag>HEAD</tag>
   </scm>
 
-  <!-- Bring some sanity to version numbering... -->
   <properties>
-    <jenkins.version>2.222.4</jenkins.version>
-    <hpi.compatibleSinceVersion>1.5.0</hpi.compatibleSinceVersion>
-    <google.api.version>1.25.0</google.api.version>
-    <google.http.version>1.21.0</google.http.version>
-    <google.guava.version>30.0-jre</google.guava.version>
-    <google-oauth.version>1.0.0</google-oauth.version>
-    <storage.revision>158</storage.revision>
     <java.level>8</java.level>
-    <concurrency>5</concurrency>
+    <jenkins.version>2.319.1</jenkins.version>
+    <hpi.compatibleSinceVersion>1.5.0</hpi.compatibleSinceVersion>
+    <google-api.version>1.33.2</google-api.version>
+    <google-http.version>1.41.4</google-http.version>
+    <google-oauth.version>1.33.1</google-oauth.version>
+    <google-storage.revision>v1-rev20211201-1.32.1</google-storage.revision>
+    <guava.version>31.0-jre</guava.version>
     <skipIts>true</skipIts>
   </properties>
 
   <dependencyManagement>
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.222.x</artifactId>
-        <version>21</version>
+        <artifactId>bom-2.319.x</artifactId>
+        <version>1148.v7261f385f859</version>
```

Comment:
do you need *this*version?  it is the latest which is known to cause issues elsewhere.  Unless a specific version is needed why not use something a little older?

Reply:
We can use an older version. changing it.